index.js

```diff
@@ -504,6 +504,9 @@ function fastifyMultipart (fastify, options, done) {
     let part
     while ((part = await parts()) != null) {
       if (part.file) {
+        if (part.file.truncated) {
+          throw new RequestFileTooLargeError()
```

Comment:
are we sure that the only condition for the file being truncated is that it is too large? Maybe it should be a different error.